packages/eslint-plugin-clarity-migration/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@clr/eslint-plugin-clarity-migration",
+  "version": "0.0.0",
+  "description": "A set of ESLint rules for Clarity migration.",
+  "main": "dist/src/index.js",
+  "typings": "dist/src/index.d.ts",
+  "keywords": [
+    "clarity",
+    "eslint-plugin",
+    "eslint"
+  ],
+  "author": "clarity",
+  "license": "MIT",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "lint": "npm run lint:src && npm run lint:test",
+    "lint:src": "tslint --project ./tsconfig.json -c ./tslint.json 'src/**/*.ts'",
+    "lint:test": "tslint -c ./tslint.json 'test/**/*.ts'",
+    "lint:fix": "npm run lint:src:fix && npm run lint:test:fix",
+    "lint:src:fix": "tslint --project ./tsconfig.json -c ./tslint.json 'src/**/*.ts' --fix",
+    "lint:test:fix": "tslint -c ./tslint.json 'test/**/*.ts' --fix",
+    "test": "jest",
```

Comment:
We use karma/jasmine for our other testing, I'm hesitant to see a new testing framework introduced. What do we get from Jest?

Reply:
I agree that we need to stick using only one framework. On the other hand, Jest is way faster than karma. In the last project with 2000+ tests, I worked on we migrated from karma to Jest and we reduced the test run time by 70%. The migration was also pretty easy. Just sharing.

Reply:
Initially, I set up the POC project tests with Jest. While transferring the plugin to this repo, I noticed that the `@clr/react` package also uses Jest and that's why I left it.

Reply:
@gnomeontherun, is it ok to leave Jest or should I swap it out for Karma+Jasmine?

Reply:
That is fair, I leave it to the team to decide but multiple options seems harder to maintain long term.